tests/Php72/Php72Test.php

```diff
@@ -64,4 +64,26 @@ public function testSplObjectId()
 
         $this->assertNull(@spl_object_id(123));
     }
+
+    /**
+     * @covers Symfony\Polyfill\Php72\Php72::sapi_windows_vt100_support
+     */
+    public function testSapiWindowsVt100Support()
+    {
+        if ('\\' !== DIRECTORY_SEPARATOR) {
```

Comment:
`@requires OSFAMILY ...` instead

Reply:
The current code matches the convention we're following, LGTM as is.

Reply:
what's behind that convention? looks pretty weird to me to not use solution provided by test runner but do it manually

Reply:
Dunno exactly, maybe just discoverability is better, so more commonly used.

Reply:
I'll do it whatever way you chaps want.

Reply:
let's keep it this way, that's consistent with the code base

Reply:
I think the reason is support for older PHPUnit versions (IIRC, `OSFAMILY` was added later)